style.css

```diff
@@ -437,4 +437,382 @@ input[type=email] {
   .fontawesome {
     display: none;
   }
+
+  .uber-navig {
+    display: flex;
+    flex-direction: column;
+    width: 345px;
+    height: 596px;
+    padding: 16px;
+    margin: 25px 24px;
+    border-radius: 16px;
+    border: 1px solid #dfe1e6;
+    box-sizing: border-box;
+    background-color: #fff;
+    justify-content: center;
+  }
+}
+
+@media screen and (min-width: 1000px) {
+  body {
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+    font-family: 'Poppins', sans-serif;
+    width: 1600px;
+  }
+
+  .header {
+    height: 280px;
+    background-color: #fff;
+    padding: 25px 155px;
+  }
+
+  #nav-bar {
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+    margin: 0 auto;
+    padding-top: 20px;
+    align-items: center;
+  }
+
+  .nav-buttons {
+    display: flex;
+    list-style-type: none;
+    cursor: pointer;
+  }
+
+  .nav-buttons a {
+    text-decoration: none;
+    height: 2em;
+    padding: 0.5em 1em;
+  }
+
+  .nav-buttons a:hover {
+    text-decoration: none;
+    height: 2em;
+    color: rgb(62, 151, 96);
+  }
+
+  main {
+    position: relative;
+    background-color: #e5e5e5;
+  }
+
+  #main-page {
+    height: 100%;
+    background-size: cover;
+    background-repeat: no-repeat;
+    width: 100%;
+    margin: 10px auto;
+    padding-bottom: 30rem;
+    padding-top: 1rem;
+  }
+
+  .header-content {
+    width: 50%;
+    margin: 0 auto;
+    padding: 0 260px;
+  }
+
+  .header-text {
+    font-size: 40px;
+    font-weight: bold;
+    line-height: 52px;
+    color: #172b4d;
+  }
+
+  .main-page-paragr {
+    width: 70%;
+  }
+
+  ul {
+    list-style: none;
+    margin: 0;
+    padding: 0;
+    display: flex;
+    gap: 8px;
+  }
+
+  .social-media ul {
+    margin: 0;
+    padding: 0;
+    display: flex;
+    gap: 15px;
+  }
+
+  .social-media ul li {
+    list-style: none;
+  }
+
+  .social-media-text {
+    color: #7f8cff;
+    font-size: 18px;
+    font-weight: 500;
+    font-family: 'Poppins', sans-serif;
+  }
+
+  .icon {
+    height: 20px;
+    width: 20px;
+  }
+
+  h1,
+  h3 {
+    color: #091e42;
+  }
+
+  h4 {
+    color: #172b4d;
+    font-weight: bold;
+    font-size: 32px;
+    line-height: 44px;
+    font-family: 'Poppins', sans-serif;
+    margin: 12px 0;
+  }
+
+  h2 {
+    color: #7f8cff;
+  }
+
+  img {
+    width: 100%;
+  }
+
+  .contact {
+    color: #f7f7f9;
+  }
+
+  #Projects {
+    margin: 0 auto;
+    background-color: #e5e5e5;
+    border-radius: 0 0 20px 0;
+  }
```

Comment:
- [ ] id is used for single elements that appear on the page for only once. Kindly, let us move our ids to classes.